packages/utilities/gel-foundations-styled-components/typography.js

```diff
@@ -0,0 +1,64 @@
+import { css } from 'styled-components';
+import MEDIA_QUERY_TYPOGRAPHY from '@bbc/gel-constants/breakpoints';
+
+export const GEL_LONG_PRIMER = css`
+  font-size: 0.9375em;
+  line-height: 1.125rem;
+  ${MEDIA_QUERY_TYPOGRAPHY.laptopAndLarger} {
+    font-size: 0.875em;
+  }
+`;
+
+/* Font styling below is a subset of BBC GEL Typography "Body Copy" */
+export const GEL_BODY_COPY = css`
+  font-size: 0.9375em;
+  line-height: 1.25rem;
+  ${MEDIA_QUERY_TYPOGRAPHY.smartPhoneAndLarger} {
```

Comment:
Didn't we change these constants? `MEDIA_QUERY_TYPOGRAPHY.SMARTPHONE_AND_LARGER`?

Reply:
Fixed